### DISCUSSION TITLE
Why is the Microsoft authentication greyed out?

### DISCUSSION THREAD

**Discussion by turbo2ltr** [requarks/wiki#6726] [Help / Questions] [answered]
First I want to confirm, my company uses the Microsoft ecosystem, I want to be able to log in with my company email using microsoft as the authenticator so any employee can log in without having to explicitly create an account in wiki.js.  I am assuming I need the Microsoft authentication module (?)
When I go to the Authentication list, Microsoft is in the list but it is greyed out and can't be selected.  Did I miss some configuration step?  The doc doesn't have a page linked for the Microsoft module.

**Comment by NGPixel** [comment, marked as answer]
Because it was never completed.
